packages/springboard/cypress/.eslintrc.json

```diff
@@ -4,5 +4,10 @@
   ],
   "env": {
     "cypress/globals": true
+  },
```

Comment:
this might be removed too

Reply:
We could, but we'd need to add them back in if we support tsx anyways.